modules/nf-core/crumble/main.nf

```diff
@@ -47,4 +47,25 @@ process CRUMBLE {
         crumble: $CRUMBLE_VERSION
     END_VERSIONS
     """
+
+    stub:
+    def args       = task.ext.args ?: ''
+    def prefix     = task.ext.prefix ?: "${meta.id}"
+    def extension  = args.contains("-O sam") ? "sam" :
+                    args.contains("-O bam") ? "bam" :
+                    args.contains("-O cram") ? "cram" :
+                    "sam"
```

Comment:
Ewww, defaulting to sam? Can we change this to default to bam?

Reply:
Good point. I have changed it to `bam` now.